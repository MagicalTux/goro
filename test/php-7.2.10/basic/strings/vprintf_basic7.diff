     *** Testing vprintf() : basic functionality - using unsigned format ***
002- 4294966185
003- int(10)
004- 4294966185 4293732729
005- int(21)
006- 4294966185 4293732729 4292621864
007- int(32)
002+ 1777777777777777775651
003+ int(22)
004+ 1777777777777777775651 1777777777777773224571
005+ int(45)
006+ 1777777777777777775651 1777777777777773224571 1777777777777767033050
007+ int(68)
     ===DONE===
