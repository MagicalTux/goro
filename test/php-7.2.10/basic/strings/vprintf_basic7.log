
---- EXPECTED OUTPUT
*** Testing vprintf() : basic functionality - using unsigned format ***
4294966185
int(10)
4294966185 4293732729
int(21)
4294966185 4293732729 4292621864
int(32)
===DONE===
---- ACTUAL OUTPUT
*** Testing vprintf() : basic functionality - using unsigned format ***
1777777777777777775651
int(22)
1777777777777777775651 1777777777777773224571
int(45)
1777777777777777775651 1777777777777773224571 1777777777777767033050
int(68)
===DONE===
---- FAILED
