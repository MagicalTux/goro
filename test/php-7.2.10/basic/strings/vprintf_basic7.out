*** Testing vprintf() : basic functionality - using unsigned format ***
1777777777777777775651
int(22)
1777777777777777775651 1777777777777773224571
int(45)
1777777777777777775651 1777777777777773224571 1777777777777767033050
int(68)
===DONE===